### ISSUE TITLE
 Server said: Could not change into directory: /var/cache/obdi 

### ISSUE THREAD

**Issue by arthurzenika** [mclarkson/obdi#20]
Following install (see #15) when trying to list add plugin I get 

`Server said: Could not change into directory: /var/cache/obdi`

**Comment by mclarkson** [comment]
Fixed by c0963c28ceda774ea23e998a3c92b55aaad0331f
Thanks!

**Comment by arthurzenika** [comment]
I'm not sure this is enough : 

```
# ls -alh  /var/cache/obdi 
ls: cannot access /var/cache/obdi: No such file or directory
```
